pkg/client/unversioned/record/event.go

```diff
@@ -222,6 +222,9 @@ func (eventBroadcaster *eventBroadcasterImpl) StartEventWatcher(eventHandler fun
 				// ever happen.
 				continue
 			}
+			if ratelimiter != nil {
+				ratelimiter.Accept()
```

Comment:
@lavalamp Maybe I don't get your point. Accept() will wait for a  token generated by token bucket algorithm.